### PR TITLE
Iris

### PR DESCRIPTION
**Pull request by inkuss**
- single server with containers was already up to date, on Iris release
- removed single server without containers (we might want keep it in Honeysuckle, but not drag it further along)
- Vagrant boxes: replaced link by the one pointing to r1-2021-release (instead of master)
- Kubernetes example: Waiting for content. For now, I added a note that one might want to look at Q3-2020 or R2-2021.

### CODE REVIEW

File: content/en/docs/Getting started/Installation/singleservernocontainers.md [hunk 1]
```diff
@@ -1,239 +0,0 @@
----
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleservernocontainers.md:1]: Can you remove the link from line 30 of content/en/docs/Getting started/Installation/_index.md to singleservernocontainers.md?

Comment (inkuss) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:1]: Done.

Comment (julianladisch) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:1]: Thanks!

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**